### ISSUE TITLE
Line/streak artifacts in projections

### ISSUE THREAD

**Issue by villekf** [GGEMS/ggems#23]
When using voxelized phantoms with enough small voxel size and large enough size, line/streak appear in the projections. While I've managed to get these appear in different cases, the example 2 can be modified easily to cause these lines (see figure): 
![image](https://github.com/GGEMS/ggems/assets/22394626/47f1d50c-0cc3-41ea-acb0-f4eb8d826e51)

The lines appear by simply increasing the size of the phantom in example 2:
`volume_creator_manager.set_dimensions(1000, 1000, 1000)`
Note that I replaced the water with air for better visibility (`box_phantom.set_material('Air')`) and used 1000000000 photons.

The center region does not have these lines and they disappear if the phantom voxel size is increased sufficiently.

Is there some way to avoid these streaks other than increasing the phantom voxel size?

**Comment by SS99aaNN** [comment]
Hello, villekf. Do you have any updates on this issue? I have encountered a similar problem and have conducted some simulations to address it. I tried using different combinations of detector crystal sizes and phantom voxel sizes to test for streak artifacts. The detector crystal material used was CSI, and the phantom material was air. The final results are as follows:

**Comment by seanlammm** [comment]
Hello, villekf. 
Have you made any progress on this issue? I have encountered a similar problem and conducted some simulations to address it. I attempted to use different combinations of detector crystal sizes and phantom voxel sizes to test for streak artifacts. The detector crystal material used was CSI, the phantom material was air, and the photon count was 1e10. The final results are as follows:
![image](https://github.com/user-attachments/assets/2c14b20c-2403-495e-8672-7ec5edccfe97)

Additionally, I conducted a simulation of a mouse phantom, attempting to scale up the crystal size, phantom size, SID, and SDD by a factor of four. For instance, the original phantom voxel size of 0.1 was scaled up to 0.4 while maintaining the same data dimensions. The final projection results showed that there were still a few streak artifacts in projections from certain angles, but these artifacts generally did not affect the reconstructed image.

![image](https://github.com/user-attachments/assets/da56f5b8-1007-4da5-bd3c-6918e8d8049b)

**Comment by villekf** [comment]
No, I haven't done any progress on this. Since I wanted to simulate a specific scanner, it was important to keep the same dimensions. Aside from the first tests, I haven't thus done any experimentation on this. Lately I've used MCGPU for any possible simulations though GGEMS does have some nice features that the MCGPU lacks.

**Comment by wenyiyang-2002** [comment]
Hello, seanlammm.
May I ask how this kind of custom mold is obtained? I mean the mouse you made. If I want to make a mammary mold, how should I operate? @seanlammm

**Comment by seanlammm** [comment]
> Hello, seanlammm. May I ask how this kind of custom mold is obtained? I mean the mouse you made. If I want to make a mammary mold, how should I operate? [@seanlammm](https://github.com/seanlammm)

I'm sorry, but I may not be able to provide useful information. This phantom was already available in the research group before I joined; it seems to have been purchased from a specialized manufacturer and was not homemade.

**Comment by wenyiyang-2002** [comment]
Ok. Thank you for your reply.
